src/Range/Range.js

```diff
@@ -1,29 +1,23 @@
-import React, { Component } from 'react'
+import React, { useEffect, useRef } from 'react'
 import PropTypes from 'prop-types'
+
 import './Range.css'
 
-export default class Range extends Component {
-  constructor(props) {
-    super(props)
-    this.minValueRef = React.createRef()
-    this.maxValueRef = React.createRef()
-  }
+export default function Range(props) {
+  const minValueRef = useRef(parseInt(props.valueMin, 10))
+  const maxValueRef = useRef(parseInt(props.valueMax, 10))
 
-  componentDidMount() {
-    this.minValueRef.current.value = parseInt(this.props.valueMin, 10)
-    this.maxValueRef.current.value = parseInt(this.props.valueMax, 10)
-  }
+  useEffect(
```

Comment:
useEffect instead of `cDM` and `CWRP`.  `CDM` is replace by the callback code in `useRef`.  `cWRP` is replace by the array of value you pass as the second argument of `useRef`.